SpatialGDK/Source/SpatialGDK/Public/Schema/UnrealMetadata.h

```diff
@@ -38,6 +38,11 @@ struct UnrealMetadata : Component
 		}
 		OwnerWorkerAttribute = GetStringFromSchema(ComponentObject, 2);
 		ClassPath = GetStringFromSchema(ComponentObject, 3);
+
+		if (Schema_GetBoolCount(ComponentObject, 4) == 1)
```

Comment:
Is it worth making spatial constants for these, to avoid magic numbers?

Reply:
Yes. That should be done in a different task I think.